### PR TITLE
Fix drag and drop in chrome when the document has iframes

### PR DESCRIPTION
**Pull request by oandregal**
Fixes https://github.com/WordPress/gutenberg/issues/6145

In Chrome, the `dragend` event won't be dispatched if an iframe is affected by the drag operation (moved as a result of dragging other nodes, etc). [Chrome bug](https://bugs.chromium.org/p/chromium/issues/detail?id=737691#c8).

## Testing

- Create a post that contains an iframe. For example, you can add an embed block (twitter, youtube video, etc).
- Drag it to several positions. 
- Drag other elements above and below it.

The expected result is that the drag-and-drop operation works and it's not stuck mid-drag.

### CODE REVIEW

**Comment by nosolosw** [comment]
Note that https://github.com/WordPress/gutenberg/pull/9311 refactors the `Draggable` component. This needs rebased once that one is merged.

File: packages/components/src/draggable/index.js
Comment (nosolosw) [inline packages/components/src/draggable/index.js]: Shall we scope the iframe search to the `.editor-block-list__layout` element? I fear that if we do that it would break at some point if we decide to change the CSS classes we use.

Also, I'm not aware of any other areas that could potentially use iframes outside the editor. Is there any? If so, this would give false positives.

Comment (nosolosw) [reply packages/components/src/draggable/index.js]: We should scope the search at least to the `#editor` element. The WordPress.com notifications bar uses an iframe.

**Comment by youknowriad** [comment]
Tested and confirmed the fix but I think design wise, it's not great. I wonder if we should show the block icon as a drag handle instead of removing the iframes which can cause weird states.

cc @jasmussen

**Comment by youknowriad** [comment]
Oh I guess the iframe removal is already in master, so I guess I'm just going to approve here and we'll see if we can improve the drag handle for blocks with iframes later.

**Review by youknowriad — APPROVED** [review]
LGTM 👍

**Comment by jasmussen** [comment]
Given designs, perhaps we should merge this along wiht the improvements that are scheduled in `add/drag-handle`?

**Pull request merged**
